routes/api/html.js

```diff
@@ -1,13 +1,7 @@
 var path = require("path");
 
 module.exports = function(app) {
-
-	app.get("/index", function(req, res) {
-		res.sendFile(path.join(__dirname, "/../public/index.html"));
-	});
-
-	// fallback use route for homepage
-	app.use(function(req, res) {
-		res.sendFile(path.join(__dirname, "/../public/home.html"));
-	});
-};
\ No newline at end of file
+  app.get("/", function(req, res) {
+    res.sendFile(path.join(__dirname, "/../public/index.html"));
+  });
+};
```

Comment:
this route isn't being used either since everything is being served from the public folder statically... I left this here though in case we want to use handlebars and need html routes moving forward though 😊